server/repositories/__tests__/cmsApi.spec.js

```diff
@@ -1,11 +1,11 @@
 const nock = require('nock');
 const {
-  primaryMenuResponse,
-  prisons,
-  landingPage,
-} = require('../../../test/resources/primaryMenu');
+  jsonApiResponse,
+  category,
+  tag,
+} = require('../../../test/resources/topics');
 const { JsonApiClient } = require('../../clients/jsonApiClient');
-const { CmsApi } = require('../cmsApi');
+const { CmsApi, topicsQuery } = require('../cmsApi');
```

Comment:
Is it worth actually putting a simple test in for `topicsQuery`? I appreciate its essentially a static constant, but wanted to ascertain if there was any value in highlighting any  fluctuations in the consistency of its value?

Reply:
it is implicitly used through nock... at one point I hardcoded the url that it generated in this test.. could potentially put that back in